README.md

```diff
@@ -21,12 +21,11 @@ The project is a CRM (customer relationship management) system for food establis
 * Track feedback on service, food, prices.
 
 
-In the architecture, we used microservices.
-This is best suited for building and deploying software applications. Each service has its own database. For communication between services sync HTTP REST used.
+In the architecture, we used microservices. Each service has its own database. For communication between services sync HTTP REST used.
```

Comment:
I suggest smth like:
The project architecture is based on microservices approach. It follows 'DB per the service' approach/pattern.
Communication between system components (services) are sync, REST is used